packages/calling-stateful-client/src/StatefulCallClient.ts

```diff
@@ -211,6 +211,12 @@ class ProxyCallClient implements ProxyHandler<CallClient> {
           return this._deviceManager;
         }, 'CallClient.getDeviceManager');
       }
+      case 'getEnvironmentInfo': {
+        return this._context.withAsyncErrorTeedToState(async () => {
+          const environmentInfo = await target.getEnvironmentInfo();
+          return environmentInfo;
```

Comment:
instead of just returning here, this result should be stored into the call context then the selector can pluck it out of the context